sqlalchemy-netezza/nzalchemy/base.py

```diff
@@ -248,13 +248,11 @@ def __init__(self, length=None, varying=False):
 
 class NVARCHAR(sqltypes.NVARCHAR):
     def __init__(self, length=None, collation=None,
-                 convert_unicode='force',
                  unicode_error=None):
         super(NVARCHAR, self).__init__(
             length,
             collation=collation,
-            convert_unicode=convert_unicode,
-            unicode_error='ignore')
+            )
```

Comment:
how to set utf8 (char_varchar_encoding) in nzpy?

Reply:
This is how we pass any nzpy related connection argument to nzalchemy:

```
import nzpy

def creator():
    return nzpy.connect(user="admin", password="password",host='localhost', port=5480, database="db1", securityLevel=0,logOptions=nzpy.LogOptions.Logfile, char_varchar_encoding='utf8')

engine = create_engine("netezza+nzpy://", creator=creator)

```